### DISCUSSION TITLE
No standard theme suitable for white background in v3?

### DISCUSSION THREAD

**Discussion by myuki** [JanDeDobbeleer/oh-my-posh#468] [Troubleshoot] [answered]
I have updated Oh My Posh from v2 to v3. But all themes don't suit white background. Even they work properly in v2. The v3 more often use white font, though they don't do that in the v2.

**Comment by JanDeDobbeleer** [comment, marked as answer]
That's because V2 used ANSI 16 colors by default (no large palettes available just yet looking at terminals at that time). However, it's rather straightforward to [create your own theme](https://ohmyposh.dev/docs/installation#override-the-theme-settings) and tweak to perfection.
